tests/test_govuk_frontend.py

```diff
@@ -138,6 +139,81 @@ def test_from_question_with_errors(self, question, snapshot):
         assert form == snapshot
 
 
+class TestCheckboxes:
+    @pytest.fixture
+    def question(self):
+        return Question(
+            {
+                "id": "yesOrNo",
+                "name": "Yes or no",
+                "question": "Yes or no?",
+                "type": "radios",
```

Comment:
Will this be `type: checkboxes` in practice? (I don't think it matters for this test)

Reply:
Oof. Thanks for catching!